app/assets/markdown/guide.md

```diff
@@ -1,24 +1,22 @@
-##What is govspeak?
+##What is Govspeak?
 
-Govspeak is a simplified 'markup' language based on [Markdown](http://daringfireball.net/projects/markdown/syntax). It's designed to be as easy-to-read and easy-to-write as possible, using simple punctuation instead of complicated tags and code.
+[Govspeak](https://github.com/alphagov/govspeak) is a simplified version of [Markdown](http://daringfireball.net/projects/markdown/syntax) used on [GOV.UK](https://www.gov.uk/). It's designed to be as easy-to-read and easy-to-write as possible, using simple punctuation instead of complicated tags and code.
 
-Govspeak was developed by the [Government Digital Service](digital.cabinetoffice.gov.uk) with some extra features that we use to format content on [GOV.UK.](https://www.gov.uk)
+Govspeak was developed by the [Government Digital Service]() with some extra features that we use to format content on [GOV.UK](https://www.gov.uk/).
```

Comment:
Ah looks like the link, `digital.cabinetoffice.gov.uk` got dropped off

Reply:
Good find! That address now seems to be a redirect to https://gds.blog.gov.uk/ so i've updated to this instead.